ui/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link name="unprocessed-script" content="lockdown.umd.js"/>
+    <link name="unprocessed-script" content="lockdown.umd.js" />
```

Comment:
It's not clear now this whitespace change is part of reorganizing the appbar and such.

I don't see any reason to rush or take short-cuts, so please make sure the change comment and the changes agree with each other.

moving whitespace / lint changes to their own commit helps.

Reply:
Will keep unrelated whitespace/lint changes in separate commits in the future as discussed

Reply:
an alternative to a separate commit is something in CONTRIBUTING.md about the coding style we're using. Or better yet: a `yarn lint:fix` script that automates it.

Reply:
Just confirmed this was caused by my prettier vscode plugin. We don't have `eslint-plugin-prettier` in this package but we use it for other dapps. I think I'll create an issue to follow up and add it so that `yarn lint-fix` will make these types of changes